volttron/platform/instance_setup.py

```diff
@@ -484,6 +484,23 @@ def do_vip():
             print('\nERROR: That address has already been bound to.')
     config_opts['vip-address'] = '{}:{}'.format(vip_address, vip_port)
 
+
+def do_instance_name():
+    global config_opts
+    # Default instance name to the vip address.
+    instance_name = config_opts.get('instance-name',
+                                    'volttron1')
+    instance_name = instance_name.strip('"')
+
+    valid_name = False
+    while not valid_name:
+        prompt = 'What is the name of this instance?'
+        new_instance_name = prompt_response(prompt, default=instance_name)
+        if new_instance_name:
```

Comment:
I believe there is more to validating an instance name than just a non-empty string.  We need to discuss this.

Reply:
JIRA issue has been submitted to discuss this further. No change at this time.